org.eclipse.m2e.jdt/src/org/eclipse/m2e/jdt/internal/UnitTestSupport.java

```diff
@@ -0,0 +1,776 @@
+/********************************************************************************
+ * Copyright (c) 2024, 2024 ptreilhes and others
```

Comment:
For the copy-right header you should use your full name (i.e. `surname lastname`).

Reply:
ok